strapi/strapi
# [DEITS] Fix available strategies in CLI

### What does it do?

- remove all strategies from the CLI
- define default values for each strategy and replace hard-coded strings with them
- rename schemasMatching -> schemaStrategy
- rename versionMatching -> versionStrategy
- added schemaStrategy `ignore`

### Why is it needed?

Cleanup for release until which strategies to expose are decided

### How to test it?

Run import with the various options available and see that they work as expected
